diff -up guvcview-src-1.5.1/src/v4l2_formats.c~ guvcview-src-1.5.1/src/v4l2_formats.c
--- guvcview-src-1.5.1/src/v4l2_formats.c~	2011-10-13 23:19:54.000000000 +0200
+++ guvcview-src-1.5.1/src/v4l2_formats.c	2011-12-19 11:39:27.900223646 +0100
@@ -22,8 +22,6 @@
 
 #include <glib/gprintf.h>
 #include <glib/gstdio.h>
-#include <glib/gutils.h>
-#include <glib/gtestutils.h>
 #include <fcntl.h>
 #include <string.h>
 #include <sys/types.h>
